packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -66,6 +67,14 @@ enum RoutePopDisposition {
   bubble,
 }
 
+/// Name for the method which is called when the system wants the current route
+/// to be shifted to the previous route in the history.
+const String popRouteMethod = 'popRoute';
```

Comment:
Can these be private?

Reply:
Thanks for the suggestion. I thought they might be useful in other methods later. You are right it is better to keep them private for now.